android/spectrumtests/build.gradle

```diff
@@ -30,6 +40,7 @@ android {
 dependencies {
     compileOnly deps.jsr305
     implementation deps.soloader
+    implementation 'androidx.multidex:multidex:2.0.1'
```

Comment:
@EwoutH  thanks for looking into this!

Can you use here deps.supportMultidex instead?

Reply:
Nice catch, that way the versions also won't get out of sync. Resolved!